packages/components/src/components/pagination/shadow.tsx

```diff
@@ -165,16 +165,20 @@ export class KolPagination implements PaginationAPI {
 					</ul>
 				</nav>
 				{this.state._pageSizeOptions?.length > 0 && (
-					<KolSelectTag
-						_hideLabel
-						_id={`pagination-size-${this.nonce}`}
-						_label={translate('kol-entries-per-site')}
-						_options={this.state._pageSizeOptions}
-						_on={{
-							onChange: this.onChangePageSize,
-						}}
-						_value={[this.state._pageSize]}
-					></KolSelectTag>
+					<div class="page-size">
+						<label htmlFor={`pagination-size-${this.nonce}`}>{translate('kol-page-per-site')}</label>
+
+						<KolSelectTag
```

Comment:
Hier müssen wir `KolSelectWcTag` verwenden und das Select-Styling in die Pagination importieren.

```suggestion
<KolSelectWcTag
	_label={translate('kol-entries-per-site')}
	_options={this.state._pageSizeOptions}
	_on={{
		onChange: this.onChangePageSize,
	}}
	_value={[this.state._pageSize]}
/>
```

Reply:
@copilot refactor suggestion